README.md

```diff
@@ -10,3 +10,20 @@ To build a new Solidus Extension, you can run
 solidus extension my_extension
 ```
 
+after you have globally installed the gem outside of your project folder:
+
+```
+gem install solidus_cmd
+```
+
+This will create a folder `solidus_my_extension` containing default content to
+let you start writing your own extension.
+
+Please update the .gemspec file with author, description, summary and email.
+Then you can run
+
+```
+bundle install
+```
+
+You can follow a tutorial in the [spree documentation](https://guides.spreecommerce.org/developer/extensions_tutorial.html#creating-an-extension)
```

Comment:
I don't think that we should reference to spree documentation from here. That documentation only apply partially for build a solidus extension.

I think will be better port that documentation to [solidus guides](https://github.com/solidusio/solidus/tree/master/guides)

Reply:
Yes, you are right.
But I think for the moment it's better than nothing, since we don't have that port at the moment.